src/routes/v1/hotelRoute.js

```diff
@@ -0,0 +1,15 @@
+import Hotel from "../../controllers/hotelController";
+import { hotelValidation } from "../../middleware/hotelMiddleware";
+import express from "express";
+
+const hotelRouter = express.Router();
+const hotel = new Hotel();
+
+hotelRouter.get("/api/hotel-reservation/", hotel.getHotel);
```

Comment:
I am not sure you went through all the feedback given from the last review, Please just go through the last review again carefully.